documentation/docs/guide/experiments/export-trained-model.md

```diff
@@ -1,9 +1,9 @@
 ---
-description: If you’re ready to share your trained model with a broader community, H2O LLM Studio allows you to export the fine-tuned model to Hugging Face with a single click.
+description: If you are ready to share your trained model with a broader community, H2O LLM Studio allows you to export the fine-tuned model to HuggingFace with a single click.
```

Comment:
```suggestion
description: If you are ready to share your trained model with a broader community, H2O LLM Studio allows you to export the fine-tuned model to Hugging Face Hub with a single click.
```

Hugging Face is the correct name of the company. We can clarify that it goes to the Hugging Face Hub.